addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1592,7 +1626,7 @@ options.registry.ThemeColors = options.registry.OptionsTab.extend({
         const supportOldColorSystem = weUtils.getCSSVariableValue('support-13-0-color-system', style) === 'true';
         const hasCustomizedOldColorSystem = weUtils.getCSSVariableValue('has-customized-13-0-color-system', style) === 'true';
         this._showOldColorSystemWarning = supportOldColorSystem && hasCustomizedOldColorSystem;
-
+        
```

Comment:
You should never have this kind of change committed. Personally, those are automatically removed when saving a file but more importantly when you commit something, you should only add the things you really wanted to modify. `git add -p .` could be of help here.

Reply:
Sorry about that. I'll make sure to not do the same mistake the next times